packages/next-wxt/env/.env.open

```
@@ -1,6 +1,6 @@
-VITE_LLM_BASE_URL=https://agent.opentiny.design/api/v1/ai/prompt
+VITE_LLM_BASE_URL=http://7.249.20.88:1234/v1
 VITE_LLM_API_KEY=sk-trial
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_LLM_API_KEY key should go before the VITE_LLM_BASE_URL key

(UnorderedKey)